### DISCUSSION TITLE
[281990] 群星 Stellaris（本体）

### DISCUSSION THREAD

**Discussion by Dew778** [pjy612/SteamManifestCache#383] [Share]
`
appid：281990
`

```
281991;f7e7dc0ebbd8af57b97ebdf6a1c60cab11601c659735dec52b75c2041c253bbd
281992;e4875742b695f223ab3bfa079d2b2e33c6785bb1e31b88e82964f1391b63066a
```
[281990_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13972688/281990_config.vdf.txt)
[[281990] 群星 Stellaris（本体）清单.zip](https://github.com/pjy612/SteamManifestCache/files/13973022/281990.Stellaris.zip)

Code：7337BCEA697F2EBE
Email:3394748293@qq.com

**Comment by ZEERDEER** [comment]
标题格式 #26

**Reply by Dew778** [reply]
已修改

**Comment by pjy612** [comment]
#26 
标题格式 
请附带清单

**Reply by Dew778** [reply]
已修改
